docs/asciidoc/static/advanced-pipeline.asciidoc

```diff
@@ -116,6 +116,14 @@ filter {
 }
 --------------------------------------------------------------------------------
 
+To set the stdout output to json format:
+[source,json]
+--------------------------------------------------------------------------------
+output {
+    stdout {codec=>rubydebug} 
```

Comment:
nope, the one that worked for me was "rubydebug"
this is what is used in the "Getting Started with Logstash Overview Video
Watch Now"

maybe json works to but I tried rubydebug